mobile/src/reducers/organization/user.reducer.ts

```diff
@@ -0,0 +1,34 @@
+import { Reducer } from 'redux';
+import { OrganizationActions } from './organization.action';
+import { User } from './user.model';
+import { Employee } from './employee.model';
+
+export interface UserState {
+    user: User;
+    employee: Employee;
+}
+
+const initState: UserState = {
+    user: null,
+    employee: null
+};
+
+export const userReducer: Reducer<UserState> = (state = initState, action: OrganizationActions) => {
+    switch (action.type) {
+        case 'LOAD-USER-FINISHED':
+            return {
```

Comment:
hint: you can use `return { ...state, user: action.user} instead. That saves you from forgetting to copy new properties when they're introduced

Reply:
Right. Fixed